Makefile

```diff
@@ -85,16 +117,29 @@ test-doc-checkout:
 	@cd doc && { \
 		git reset --hard ; \
 		git fetch --all ; \
-		[ $$(git branch --show-current) != master ] && git checkout master || : ; \
-		[ ${branch} != "master" ] \
+		[ "$$(git branch --show-current)" != "master" ] && git checkout master || : ; \
+		[ "${branch}" != "master" ] \
```

Comment:
> I am not sure I understand the added value of quotes when all these values can by definition contain no space.

Reply:
It is a bug actually, somehow some shells like mine will complain if no quotes are added —just one more reason to move this to Python or something more cross-platform.

I can extract this to another PR, it is standalone.

Reply:
Nah don't worry about it, the administrative costs of extraction are probably not worth it 😉